conf_collectd_plugin.py

```diff
@@ -0,0 +1,375 @@
+import sys
+import re
+
+import install_utils as utils
+import config
+
+COLLECTD_HOME = "/etc/collectd"
+COLLECTD_CONF_DIR = COLLECTD_HOME+"/managed_config"
+
+# http response header
+NOT_AUTH = 2
+NOT_FOUND = 1
+HTTP_OK = 0
+INVALID_URL = -1
```

Comment:
you could possibly just reuse the well known HTTP response codes instead of translating?
ie NOT_AUTH is 401, NOT_FOUND is 404, OK is 200, etc?